### PR TITLE
sched ; refactor scheduler feature

### PR DESCRIPTION
**Pull request by joohyun0115**
Integration of distributed scheduler files in the guest.c.
If you want to develop a scheduler, at that time,
you can create file.(scheduler, policy...)

### CODE REVIEW

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/108/

**Comment by in1004kyu** [comment]
I thinks it is good~

File: hypervisor/guest.c [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+static void guest_sched_start()
+{
+    /* Switch to the first guest */
+    context_switch_to_initial_guest();
+}
+
+void guest_switch_to_next_guest(void *pdata)
+{
+    struct arch_regs *regs = pdata;
+#if 0 /* ignore message due to flood log message */
```
Comment (Wonseok) [inline hypervisor/guest.c]: Please remove unnecessary codes. You might want to extract this function outside.

File: hypervisor/guest.c [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    /* Begin running test code for newly implemented features */
+    hvmm_tests_main();
+
+    /* Print Banner */
+    printH("%s", BANNER_STRING);
+
+    /* Switch to the first guest */
+    guest_sched_start();
+
+    /* The code flow must not reach here */
```
Comment (Wonseok) [inline hypervisor/guest.c]: The code can not be reach here.

**Comment by Wonseok** [comment]
Please take a look.

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/111/

**Comment by kesl** [comment]
CI : Test FAILed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/112/

**Pull request closed without merging**